diff --git a/dist/chunk-4CPDX3L5.js b/dist/chunk-4CPDX3L5.js
index feb82b5f3a3739fffa6a2b1905ccaa059043c9cf..2cc9735a456f92df1f8c7eed4c9607f0f01d8f1e 100644
--- a/dist/chunk-4CPDX3L5.js
+++ b/dist/chunk-4CPDX3L5.js
@@ -39293,10 +39293,11 @@ var watch = async (config) => {
   const { root: root4, collections, configImports } = config;
   logger.info(`watching for changes in '${root4}'`);
   const patterns = Object.values(collections).flatMap(({ pattern }) => pattern);
-  const watcher = watch2([".", ...configImports], {
+  const watcher = watch2([...patterns, ...configImports], {
     cwd: root4,
     ignoreInitial: true,
-    awaitWriteFinish: { stabilityThreshold: 50, pollInterval: 10 }
+    awaitWriteFinish: { stabilityThreshold: 50, pollInterval: 10 },
+    ignored: ['**/node_modules/**', '**/.git/**', '**/dist/**', '**/.next/**', '**/.velite/**', '**/.turbo/**']
   }).on("all", async (event, filename) => {
     if (event === "addDir" || event === "unlinkDir") return;
     if (filename == null || typeof filename !== "string") return;
