diff --git a/dist/main.js b/dist/main.js
index ccf821f0a501198b663edc6f205b1532122ff8f9..57d4ad20f26d99e4a8e62778b235a60597476803 100644
--- a/dist/main.js
+++ b/dist/main.js
@@ -41,6 +41,14 @@ const load = async (tsRelativePath, options) => {
     return loaded;
 };
 exports.load = load;
+var _invalidateRequireCacheForFile = function (filePath) {
+    delete require.cache[require.resolve(filePath)];
+  };
+  
+  var requireNoCache = function (filePath) {
+    _invalidateRequireCacheForFile(filePath);
+    return require(filePath);
+  };
 const loadSync = (tsRelativePath, options) => {
     if (options === null || options === void 0 ? void 0 : options.allowConfigurationWithComments) {
         const commentConfig = commentParser.getTsImportCommentConfigSync(tsRelativePath);
@@ -62,12 +70,12 @@ const loadSync = (tsRelativePath, options) => {
     catch (err) {
     }
     if (jsFileExists && !utils.isFileNewer(tsFileExists, jsFileExists)) {
-        const loaded = require(jsPath);
-        return loaded;
+    const loaded = requireNoCache(jsPath);
+    return loaded;
     }
     providers.loadSync(Object.assign({ tsPath,
         jsPath }, config));
-    const loaded = require(jsPath);
+    const loaded = requireNoCache(jsPath);
     return loaded;
 };
 exports.loadSync = loadSync;